automod2/automod2.py

```diff
@@ -594,7 +628,7 @@ def cleanup(self):
             channels = server['channels']
             for channel_id in list(channels.keys()):
                 channel = channels[channel_id]
-                if channel.get('whitelist') or channel.get('blacklist') or channel.get('image_limit'):
+                if [channel.get(_) for _ in ['whitelist', 'blacklist', 'image_limit', 'auto_emoji']]:

```

Comment:
_ is used to indicate that you're not using an item, and you're using it, so use an actual variable name.

Also I think this logic is wrong. An array is truthy if it has any contents, even None, so this will always evaluate to true.

Reply:
oh right originally evo gem was a list and then i changed stuff

Reply:
jk this is not even the right file uh nvm ignore me

Reply:
fixed